pulp_container/app/serializers.py

```diff
@@ -159,6 +159,13 @@ class BlobSerializer(SingleArtifactContentSerializer):
 
     digest = serializers.CharField(help_text="sha256 of the Blob file")
 
+    def __init__(self, *args, **kwargs):
+        """Fix for bindings to allow for on-demand blobs."""
+        # TODO: Move into pulpcore
+        super().__init__(*args, **kwargs)
+        if "artifact" in self.fields:
+            self.fields["artifact"].allow_null = True
+
```

Comment:
So if I read correctly, this allows to serialize on-demand blobs.
Is this serializer ever used to create blobs via the api? It may be OK for this piece of content, but i doubt, we want it generally for serializers that are also used to create content.

Reply:
Can you just add these last two sentences as comments?